packages/bot-skeleton/src/scratch/blocks/Math/math_on_list.js

```diff
@@ -63,13 +63,13 @@ Blockly.JavaScript.math_on_list = block => {
 
     if (operation === 'SUM') {
         list = Blockly.JavaScript.valueToCode(block, 'LIST', Blockly.JavaScript.ORDER_MEMBER) || '[]';
-        code = `(${list} || [0]).reduce(function(x, y) { return x + y; })`;
+        code = `([${list}] || [0]).reduce(function(x, y) { return x + y; })`;
```

Comment:
if `list = Blockly.JavaScript.valueToCode(block, 'LIST', Blockly.JavaScript.ORDER_MEMBER) || '[]';`
returns a array then in the second line wouldn't it become [[list]], and if blockly returns nothing then it would become [[]] ?
`([${list}] || [0]).reduce(function(x, y) { return x + y; })`

Can you please explain? I'm bit confused

Reply:
List is an object, when we don't put brackets around the input value, if the user enter a single number instead of a list, then something like this happen: (5).reduce() which is not correct and causes an issue.

Reply:
Okay, can we do this then?

`list = Blockly.JavaScript.valueToCode(block, 'LIST', Blockly.JavaScript.ORDER_MEMBER) || 0;`
``code = `([${list}]).reduce(function(x, y) { return x + y; })``;